challenges/day5-issues/infinite-scroll/gallery.html

```
@@ -0,0 +1,15 @@
+# Infinite Scroll Gallery (Vanilla JS)
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)